src/graphql/local.ts

```diff
@@ -1,15 +1,21 @@
 import { graphql } from "gql.tada";
 import { LOCAL_CONNECTION_ID } from "../constants";
-import { ContextValues, setContextValue } from "../context";
 import { Logger } from "../logging";
 import { LocalKafkaCluster } from "../models/kafkaCluster";
+import { LocalSchemaRegistry } from "../models/schemaRegistry";
 import { getSidecar } from "../sidecar";
 import { createLocalConnection, getLocalConnection } from "../sidecar/connections";
 
 const logger = new Logger("graphql.local");
 
-export async function getLocalKafkaClusters(): Promise<LocalKafkaCluster[]> {
-  let localKafkaClusters: LocalKafkaCluster[] = [];
+export interface LocalResourceGroup {
```

Comment:
This could be respelled to something `LocalEnvironmentGroup`-like, but we need to get through https://github.com/confluentinc/vscode/issues/435 first